### ISSUE TITLE
CoRD Crashes on Sierra

### ISSUE THREAD

**Issue by JacquesCil** [dorianj/CoRD#81]
Hi, I am in dire need of assistance with CoRD, on of my MAC's at the office the app keeps crashing, no matter what I do, I have deleted the plist files, readded the servers etc. Please assist

**Comment by leftyftw** [comment]
I don't believe anyone is maintaining this project anymore.  Microsoft's Beta RDP client is actually quite useful.

**Comment by lemonmojo** [comment]
Also see https://github.com/dorianj/CoRD/issues/74

**Comment by dorianj** [comment]
> I don't believe anyone is maintaining this project anymore. Microsoft's Beta RDP client is actually quite useful.

Correct -- i'm not maintaining this any longer :(

**Comment by dukechem** [comment]
Both CoRD and Microsoft's RDP crash a lot on me with 10.11, but on 10.10 were mostly ok.
I'm trying "Royal TSX" which I have set as default app for *.rdp files and used for the first time in past 5 minutes (Had to install the "Remote Desktop plugin (based on FreeRDP"). In the remote session, if I try to open volumes on my mac, it says "attempt to access invalid address", but that is a feature for me: I usually disabled sharing my mac volumes (as mapped drives) on remote windows pc. I'll follow up with more info after I've used it more than 5 minutes ;-)  I first heard of it at:
[http://apple.stackexchange.com/questions/208732/replacement-for-cord-microsoft-rdp](http://apple.stackexchange.com/questions/208732/replacement-for-cord-microsoft-rdp) said:

"I've found, and am now loving, Royal TSX. Not only does it do exactly what I loved CoRD for - multiple RDP connections in a single window - it has a plugin system that'll connect to VNC, TeamViewer, web pages etc, and more in the future. The RDP plugin is based on FreeRDP, and seems to support the latest RDP protocol enhancements.

Some of the functionality requires the companion server product, and it'll only do 10 connections in "free" mode ($29 for unlimited connections). But given that it's a stable, feature-rich client that is being actively developed, I can recommend it highly."

**Comment by lemonmojo** [comment]
@dukechem For Royal TSX support, please [contact us](https://www.royalapplications.com/go/support).
